src/physics/cam/clubb_mf.F90

```diff
@@ -1027,6 +1040,59 @@ subroutine knuth(kiss_gen,lambda,kout)
 
   end subroutine knuth
 
+  subroutine hormann(kiss_gen,lambda,kout)
+  !**********************************************************************
+  ! Discrete random poisson
+  ! Implements PTRS algorithm from W. Hormann

```

Comment:
What does PTRS stand for? If it's not too long, I'd like to see the unabbreviated version here.